### DISCUSSION TITLE
怎么查看在本地集成哪些cmake的第三方库

### DISCUSSION THREAD

**Discussion by yelibumeng** [xmake-io/xmake#3300] [Q&A]
怎么查看在本地集成了哪些cmake的第三方库

**Comment by waruqi** [comment]
? 没懂你的意思，自己本地集成了哪些库 自己不知道？

xmake-repo 仓库中的包，可以 xrepo search 搜，本地安装过的包，可以 xrepo scan 扫

**Reply by yelibumeng** [reply]
![image](https://user-images.githubusercontent.com/62092572/212794315-d0db5c0d-b1f8-4ebc-aaa7-315667477f9f.png)
通过这种方式的集成的呢？大佬

**Reply by waruqi** [reply]
你自己集成的哪些包，自己不知道么？还要查看什么

**Reply by yelibumeng** [reply]
就是想扫一遍，哪些是走的本地集成，有些是走了远程
